framework/util/platform.h

```diff
@@ -288,28 +293,49 @@ inline void* GetProcAddress(LibraryHandle handle, const char* name)
     return dlsym(handle, name);
 }
 
+#if defined(__ANDROID__)
+static const prop_info* GetAndroidPropertyInfo(const char* name)
+{
+    // According to system_properties.h "Property lookup is expensive, so it can be useful to cache the result of this
+    // function.". For that reason we cache property infos.
+    static std::unordered_map<std::string, const prop_info*> property_name_info_cache;
+
+    const std::string prop_name = std::string(name);
+    const prop_info*  pi;
+
+    auto prop_name_info = property_name_info_cache.find(prop_name);
```

Comment:
is this a problem in respect to multi-threading, or is this code never used concurrently?